.github/workflows/check.yml

```diff
@@ -25,14 +25,14 @@ jobs:
       MESON_EXTRA_OPTS: "-Ddpdk:platform=generic"
       DEBIAN_FRONTEND: noninteractive
       NEEDRESTART_MODE: l
-      CC: gcc-14
+      CC: ccache gcc-14
```

Comment:
IIRC, meson detects ccache presence and appends it automatically to the compiler.
So you don't need to set ccache in the CC: variable.

Reply:
I copy pasted this from the dpdk GitHub workflow. I will try removing it.